index.bs

```diff
@@ -112,8 +114,24 @@ The Magnetometer Interface {#magnetometer-interface}
   };
 </pre>
 
-To <dfn>Construct a Magnetometer Object</dfn> the user agent must invoke the
-<a>construct a Sensor object</a> abstract operation.
+
+The RawMagnetometer Interface {#raw-magnetometer-interface}
```

Comment:
I suggest you open an issue to investigate the higher level `MagneticFieldSensor`.

The key use case for `RawMagnetometer` is mobile VR and its magnetic input.